src/Security/CasAuthenticator.php

```diff
@@ -80,8 +80,13 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
         return new RedirectResponse($request->getUri());
     }
 
-    public function onAuthenticationSuccess(Request $request, TokenInterface $token, string $firewallName): Response
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, string $firewallName): ?Response
```

Comment:
That makes sense and the interface is still fullfilled: https://github.com/symfony/security-http/blob/9bac2f730fec70e89f395feb9d30cfba2b500771/Authenticator/AuthenticatorInterface.php#L75